lib/Containers/NodeHashMap.h

```diff
@@ -33,4 +37,12 @@ template<class K, class V,
              typename absl::node_hash_map<K, V, Hash, Eq>::allocator_type>
 using NodeHashMap = absl::node_hash_map<K, V, Hash, Eq, Allocator>;
 
+namespace pmr {
+template<class K, class V,
+         class Hash = typename absl::node_hash_map<K, V>::hasher,
+         class Eq = typename absl::node_hash_map<K, V, Hash>::key_equal>
+using NodeHashMap =
+    absl::node_hash_map<K, V, Hash, Eq,
+                        std::pmr::polymorphic_allocator<std::pair<const K, V>>>;
+}
```

Comment:
Why do we need this using globally?
Same about other changes in the Containers directory

Reply:
why not? There are other places where we can make use of pmr version of NodeHashMap.

Reply:
For what task?